game.go

```diff
@@ -14,26 +14,52 @@ const (
 type Game struct {
     board [][]uint8
     currentMove Player
+    gameBoardSize int //added board size variable for testing out of scope moves
 }
 
 func NewGame(boardSize int) Game {
     board := make([][]uint8, boardSize)
     for i := range board {
         board[i] = make([]uint8, boardSize)
     }
-    newGame := Game{board: board, currentMove: DARK}
+    board = boardStars(board, boardSize)
+    newGame := Game{board: board, currentMove: DARK, gameBoardSize: boardSize}
     return newGame
 }
 
+func boardStars(board [][] uint8, boardSize int) [][] uint8 {
+    center := boardSize/2
+    if boardSize < 12 {
+        board [center][center] = 8
+    } else if (boardSize > 12 && boardSize < 18) {
+        board [3][3] = 8
+        board [3][boardSize - 4] = 8
+        board [boardSize - 4][3] = 8
+        board [boardSize - 4][boardSize - 4] = 8
+        board [center][center] = 8
+    } else if (boardSize > 18) {
+        board [3][3] = 8
+        board [3][center] = 8
+        board [3][boardSize - 4] = 8
+        board [center][3] = 8
+        board [center][center] = 8
+        board [center][boardSize - 4] = 8
+        board [boardSize - 4][3] = 8
+        board [boardSize - 4][center] = 8
+        board [boardSize - 4][boardSize - 4] = 8
+    }
+    return board
+}
+
 func (game *Game) printGame() {
-    row := 0
+    row := 1 //edited to start at 1, so player labels start at 1, and so 0 can be used for errors in entering rows/columns
```

Comment:
nit:
good change, but I would remove the comment

Reply:
agreed, it was just there for you but i guess you got to see it in the pull request haha